src/backend/distributed/transaction/multi_shard_transaction.c

```diff
@@ -176,8 +176,12 @@ BeginTransactionOnShardPlacements(uint64 shardId, char *userName)
 		 */
 		MarkRemoteTransactionCritical(connection);
 
-		/* issue BEGIN */
-		RemoteTransactionBegin(connection);
+		/* the special BARE mode (for e.g. VACUUM/ANALYZE) skips BEGIN */
```

Comment:
Forgettable nitpick: Replace VACUUM/ANALYZE with "commands that can't be run in transactions (like ...)" or such?